packages/types/v1/webhooks.ts

```diff
@@ -18,6 +19,7 @@ export const ZWebhookInput = z.object({
   url: z.string().url(),
   name: z.string().nullish(),
   triggers: z.array(ZPipelineTrigger),
+  source: z.enum(["user", "zapier"]).nullish(),
```

Comment:
same here. I think making it `optional` for the input is the best choice as we will use the default `user` value then.